### ISSUE TITLE
Different behaviour on userdata-bbb_listen_only_mode=false on 2.2 and 2.3

### ISSUE THREAD

**Issue by galupa** [bigbluebutton/bigbluebutton#12681]
**Describe the bug**
Not sure if this is a bug or not, but I found a bit different behavior on BBB version 2.2 and 2.3 when set up `userdata-bbb_listen_only_mode=false`

**To Reproduce**
1. Join a room with userdata-bbb_listen_only_mode=false ```e.q. https://bbb.x.x/bigbluebutton/api/join?createTime=1624935759459&fullName=123&join_via_html5=true&meetingID=gztpzdefqynvo9d6mhl2bge85ixsrktqjnikltig&password=gibQwoNVfFmm&userID=gl-iqfotrjdxvjd&userdata-bbb_listen_only_mode=false&userdata-bbb_skip_check_audio_on_first_join=falsechecksum=e31039df5f144095d54bfd0a56be0d8da751a12c```
2. Or set up `userdata-bbb_listen_only_mode=false` on BBB application parameter and join a room

**Expected behavior**
On BBB 2.2, the user is automatically join microphone without choosing anything.

**Actual behavior**
On BBB 2.3 there is still a pop up shows when joining the room even if it is only microphone.

**Screenshots**
On BBB 2.3, it shows like this:
![Screenshot from 2021-06-29 14-56-13](https://user-images.githubusercontent.com/29434066/123731210-d2c56c80-d8eb-11eb-9a30-1063a35616b3.png)

While on 2.2 the pop up does not shows and automatically joining the mic.

**Desktop (please complete the following information):**
 - OS: Ubuntu Bionic
 - Browser: Firefox (89.0.1)

**Additional context**

**Comment by mailalaki** [comment]
Yes, unfortunately it is. An unnecessary and annoying step has been added here!
@ffdixon @antobinary , Please review this issue. 🙏

**Comment by mailalaki** [comment]
@ffdixon @antobinary

**Comment by antobinary** [comment]
Should be resolved as part of BBB 2.4.1
